RastenbergerRest/src/main/java/com/amt/rastenbergerrest/resources/FoodOfferResource.java

```diff
@@ -95,9 +97,11 @@ public Response deleteFoodOffer(@Context UriInfo uriinfo, @PathParam("offerId")
     }
 
     // Add sub resource
+
     @Path("{offerId}/participants")
     public ParticipantResource getParticipantResource() {
-        return new ParticipantResource();
```

Comment:
I think this line was the problem. It seems by creating the object manually the cdi framework wasn't used at all for creating the subsequent objects. I now just use cdi at member attribute level. Maybe there is a way to inject the ParticipantResource directly into the method.